src/screens/SUE/SueReportScreen.tsx

```diff
@@ -650,5 +687,8 @@ const styles = StyleSheet.create({
   },
   contentContainer: {
     width: device.width
+  },
+  noPaddingTop: {
+    paddingTop: 0
```

Comment:
this makes the divider to close to the buttons on the next screens.

<img width="358" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/d87d163a-897c-49fb-91b2-49bff2313100">

we need the padding like before.

Reply:
That's right! I missed that. I've updated it now. Now there is `noPaddingBottom` instead of `noPaddingTop`. So it will appear correctly after step 2. 👍